condDatasetSubmitter.py

```diff
@@ -704,6 +710,14 @@ def printInfo(options):
 #-------------------------------------------------------------------------------
 
 if __name__ == "__main__":
+  #Raise an error if couchID files exist
+  p = subprocess.Popen("ls", stdout=subprocess.PIPE, shell=True)
+  out = p.stdout.read().strip()
+  newlist = out.split('\n')
+  substring = ".couchID"
+  for object in newlist:
+    if substring in object:
+      raise ValueError("couchID file exists, please remove it");
```

Comment:
Ciao @mmusich 
it's a matter of stye philosophy: I prefer that if there's a probelm, the user is warned and the program stops - rather than the program taking file removal actions w/o the consent of the user. 
As a user, I would like to be in control of which file I remove when (for instance: you may want to rename or move elsewhere the . couchID files).
cheers, g.